pick_trajectory: [
  {position: [0.7, 0.2, 0.7],       orientation: [0.3, 0.4, 0.4, 0.5],  time: 5,  nsteps: 50},
  {position: [0.5, -0.505, 0.4],    orientation: [0, 1, 0, 0],          time: 5,  nsteps: 50},
  {position: [0.5, -0.505, 0.195],  orientation: [0, 1, 0, 0],          time: 1,  nsteps: 50}
]

place_trajectory: [
  {position: [0.5, -0.505, 0.3],  orientation: [0, 1, 0, 0],        time: 1,  nsteps: 50},
  {position: [-0.3, -0.2, 0.5],   orientation: [0.5, 0.5, 0, 0.5],  time: 5,  nsteps: 50},
  {position: [-0.5, 0.1, 0.3],    orientation: [1, 0, 0, 0],        time: 5,  nsteps: 50}
]